src/ProjectModule/Components/ProjectForm/ProjectForm.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react'
+import styles from './ProjectForm.module.css'
+import { useNavigate, useParams } from 'react-router-dom'
+import { useForm } from 'react-hook-form';
+import axios from 'axios';
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { toast } from 'react-toastify';
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+
+
+export default function ProjectForm() {
+    const navigate=useNavigate();
+    const {register,handleSubmit,formState:{errors}}=useForm();
+    const token=localStorage.getItem('userToken');
+    const {action}=useParams();
+    const [currentProject,setCurrentProject]=useState(null);
+    const curruntProjectId=localStorage.getItem('curruntProjectId');
+    const [isLoading, setIsLoading] = useState(false);
+
+    const onSubmitAdd= async(data:any)=>{
+        setIsLoading(true);
+        // console.log('Add');
+        // console.log(data);
+        try {
+            let response=await axios.post(`${baseUrl}/Project`,data,{headers:{Authorization:token}});
+            console.log('add project response ',response);
+            toast.success('Project Added Successfuly');
+            navigate('/dashboard/project-list');
+        } catch (error) {
+            setIsLoading(false);
+            console.log(error);
+            toast.error(error?.response?.data?.message||'Somthing went wrong!');
+        }
+        
+    }
+
+    const onSubmitUpdate= async(data:any)=>{
```

Comment:
We should avoid using any and try our best to write the correct type for the data.